sdcm/tester.py

```diff
@@ -356,14 +356,23 @@ def __init__(self, *args, **kwargs):  # pylint: disable=too-many-statements,too-
 
     @cached_property
     def argus_test_run(self):
-        try:
-            argus_test_run = ArgusTestRun.from_sct_config(test_id=UUID(self.test_id),
-                                                          sct_config=self.params)
-            argus_test_run.save()
-            return argus_test_run
-        except Exception as exc:  # pylint: disable=broad-except
-            self.log.warning("Unable to set up Argus connection: %s", exc.args[0])
-            self.log.debug("Error details: ", exc_info=True)
+        if self.params["enable_argus"]:
```

Comment:
IMHO this should be `self.params.get("enable_argus")`, but this is not a major issue, as it is set in the `test_default` configuration

Reply:
Agree, better to use "get" method.

Reply:
Changed it :slightly_smiling_face: